src/code/game.c

```diff
@@ -159,7 +160,7 @@ void GameState_InitArena(GameState* gameState, size_t size) {
     __assert("../game.c", 1035);
 }
 
-void GameState_Realloc(GameState* gameState, size_t size) {
+void_dbg_hungupte_Realloc(GameState* gameState, size_t size) {
```

Comment:
Hmm, something seems to have gone wrong in this file 🤔.

Reply:
wtf, how